features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin_new/source/src/app/components/Throttling/Blacklist/AddEdit.jsx

```diff
@@ -115,12 +115,12 @@ function AddEdit(props) {
     const {
         conditionType, conditionValue, conditionValue: {
             endingIp, startingIp, invert, fixedIp,
-        },
+        }, conditionStatus,
     } = state;
     const restApi = new API();
 
     const onChange = (e) => {
-        if (e.target.name === 'invert') {
+        if (e.target.name === 'invert' || e.target.name === 'conditionStatus') {
```

Comment:
Shall we destructure the `name` property from the event and use it here, Otherwise we are repeating the `e.target` here

Reply:
Fixed with `da72cb5`